src/mmw/apps/bigcz/clients/cuahsi/search.py

```diff
@@ -235,6 +238,12 @@ def make_request(request, expiry, **kwargs):
 
 
 def get_services_in_box(box):
+    try:
+        client = Client(CATALOG_URL, timeout=settings.BIGCZ_CLIENT_TIMEOUT)
```

Comment:
My recollection is that the `Client` initialization is a somewhat expensive process (which makes sense now that we see it is making HTTP requests).  Would it make sense to keep the variable scoped to the module, but initialize it conditionally inside the view function?  I'm not sure how much of an impact this has on a search request response time.